test/integ/workflowapi/test_workflow_crud.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+###############################################################################
+# Integration testing for the MIE workflow API
+#
+# PRECONDITIONS:
+# MIE base stack must be deployed in your AWS account
+#
+# Boto3 will raise a deprecation warning (known issue). It's safe to ignore.
+#
+# USAGE:
+#   cd tests/
+#   pytest -s -W ignore::DeprecationWarning -p no:cacheprovider
+#
+###############################################################################
+
+import urllib3
+import time
+
+# local imports
+import validation
+
+
+def test_workflow_api(workflow_api, api_schema, operation_configs, stage_configs, workflow_configs):
+    api = workflow_api()
+    schema = api_schema
+    urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
I know this is from the original tests, but do you know why this is needed?

Reply:
From what I understand, the default behavior of urllib3 is to attempt certificate validation on all https requests. I'm not positive on the specifics though. 

What we're doing here is basically saying to ignore certificate validation warnings (insecure requests) on these https requests because it's always going to generate warnings as we're not providing the relevant certificate(s) to validate. 

I think this is acceptable behavior for tests. If we were passing along a confidential or high risk payload I'd have a different take on this.